net/unbound/Makefile

```diff
@@ -115,6 +117,7 @@ CONFIGURE_ARGS += \
 	--enable-allsymbols \
 	--enable-tfo-client \
 	--enable-tfo-server \
+	$(if $(CONFIG_OPENSSL_WITH_EC),--enable-ecdsa,--disable-ecdsa) \
```

Comment:
Someone need to send a patch to enable `OPENSSL_WITH_EC` by default (pointing to openvswitch/unbound issues + showing size increase) then add a dependency on `@OPENSSL_WITH_EC`

Reply:
https://github.com/openwrt/openwrt/blob/master/package/libs/openssl/Config.in#L5
OPENSSL_WITH_EC already default to y, we indeed have buildroot issues here

Reply:
See https://github.com/openwrt/openwrt/commit/dd9da5146299b769fad874d32a8ae110312cb68f for an explanation of the issue, and an ugly workaround that I would like to use for a few options as possible.

Reply:
I guess rather than implicitly disabling ECDSA, making it an explicit UNBOUND_WITH_ECDSA config symbol depending on OPENSSL_WITH_EC, and defaulting to y, would make sense.

We can also add an @OPENSSL_WITH_EC dependency, but this will not fix the build issue when unbound is built without being enabled in .config. I do have a few ideas how we could fix our dependency system to allow to express such dependencies, but it would involve another major revamp of our Kconfig generator...

Reply:
This is also breaking builds of stubby which is holding back testing updates.